package.json

```diff
@@ -50,16 +50,16 @@
   "dependencies": {
     "@babel/core": "^7.0.0",
     "@expo/samples": "~3.0.3",
-    "@expo/vector-icons": "^10.0.3",
-    "@react-native-community/masked-view": "^0.1.7",
+    "@expo/vector-icons": "^10.0.0",
+    "@react-native-community/masked-view": "0.1.6",
```

Comment:
Should probably update these

Reply:
Can probably be fixed after #74 is merged to master, and this branch is rebased onto master